### ISSUE TITLE
Node.js  Loaders Team Meeting 2022-03-29

### ISSUE THREAD

**Issue by mhdawson** [nodejs/loaders#72]
## Time

**UTC Tue 29-Mar-2022 18:00 (06:00 PM)**:

| Timezone      | Date/Time             |
|---------------|-----------------------|
| US / Pacific  | Tue 29-Mar-2022 11:00 (11:00 AM) |
| US / Mountain | Tue 29-Mar-2022 12:00 (12:00 PM) |
| US / Central  | Tue 29-Mar-2022 13:00 (01:00 PM) |
| US / Eastern  | Tue 29-Mar-2022 14:00 (02:00 PM) |
| EU / Western  | Tue 29-Mar-2022 19:00 (07:00 PM) |
| EU / Central  | Tue 29-Mar-2022 20:00 (08:00 PM) |
| EU / Eastern  | Tue 29-Mar-2022 21:00 (09:00 PM) |
| Moscow        | Tue 29-Mar-2022 21:00 (09:00 PM) |
| Chennai       | Tue 29-Mar-2022 23:30 (11:30 PM) |
| Hangzhou      | Wed 30-Mar-2022 02:00 (02:00 AM) |
| Tokyo         | Wed 30-Mar-2022 03:00 (03:00 AM) |
| Sydney        | Wed 30-Mar-2022 05:00 (05:00 AM) |

Or in your local time:
* https://www.timeanddate.com/worldclock/fixedtime.html?msg=Node.js+Foundation+Loaders%20Team+Meeting+2022-03-29&iso=20220329T1800
* or https://www.wolframalpha.com/input/?i=06PM+UTC%2C+Mar+29%2C+2022+in+local+time

## Links

* Minutes Google Doc: <https://docs.google.com/document/d/1Bh34ZqUdRPHD9z8RTOh2EU-juwNM1Gh8zex-cV1Kn6s/edit>

## Agenda

Extracted from **loaders-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.

## Invited

* Loaders team: @nodejs/loaders

## Notes

The agenda comes from issues labelled with `loaders-agenda` across **all of the repositories in the nodejs org**. Please label any additional issues that should be on the agenda before the meeting starts.

## Joining the meeting

* link for participants: <https://zoom.us/j/97506450082>

* For those who just want to watch: <https://www.youtube.com/c/nodejs+foundation/live>

**Comment by GeoffreyBooth** [comment]
No agenda, is there anything to meet about?

**Comment by JakobJingleheimer** [comment]
Anything from @arcanis for ambient loaders? (I think he had some action items from the previous meeting?) Otherwise, nothing I can think of (chaining implementation isn't ready / needing discussion yet).

**Comment by arcanis** [comment]
On my side I'll merge the ambient loader RFC and open a followup issue to discuss async whether ambient loaders should just be the default behavior of all loaders.

I haven't opened issues about reentrency just yet because I'd like to experiment with ambient loaders first, which is blocked on having an implementation, which itself is pending chaining implementation

**Comment by GeoffreyBooth** [comment]
That all sounds good. I'll cancel this one.
